Antidote/ActiveCallViewController.m

```diff
@@ -223,10 +239,20 @@ - (void)installConstraints
         make.top.equalTo(self.topViewContainer.topMargin);
     }];
 
-    [self.tableViewOfPausedCalls makeConstraints:^(MASConstraintMaker *make) {
+    [self.pauseCallsContainer makeConstraints:^(MASConstraintMaker *make) {
         make.top.equalTo(self.topViewContainer.bottom);
         make.width.equalTo(self.view);
-        make.bottom.equalTo(self.view.bottom).with.offset(-kTableViewBottomOffSet);
+        make.bottom.equalTo(self.view);
```

Comment:
Hm, why you want to place this view under `topViewContainer`? Top container is "outside" of tableView too. I think you can do `make.edges.equalTo(self.view);` here